README.md

```diff
@@ -11,38 +13,67 @@ A python package for music and audio analysis.
 [![codecov](https://codecov.io/gh/librosa/librosa/branch/main/graph/badge.svg?token=ULWnUHaIJC)](https://codecov.io/gh/librosa/librosa)
 [![Docs](https://github.com/librosa/librosa/actions/workflows/docs.yml/badge.svg)](https://github.com/librosa/librosa/actions/workflows/docs.yml)
 
-Documentation
--------------
+ ##  Table of Contents
+
+**[Documentation](#Documentation)**<br>
+**[Installation](#Installation)**<br>
```

Comment:
It seems like bad form to mix HTML (`<br>`) and markdown.  Is there another way to accomplish this?